routes/options.js

```diff
@@ -124,6 +131,21 @@ function validate(key, value) {
         return value;
     } else if (key == "cooldown_time" && parseFloat(value) >= 0) {
         return parseFloat(value).toString();
+    } else if (key == "notif_time_threshold" && parseFloat(value) >= 0) {
+        return parseFloat(value).toString();
+    } else if (key == "notif_time_interval" && parseFloat(value) >= 0) {
+        return parseFloat(value).toString();
+    }
+}
+
+// Creates an update message for float option fields
+function create_float_update_message(field_name, prev_value, value) {
+    if (parseFloat(prev_value) > 0 && parseFloat(value) > 0) {
+        return field_name + " time limit updated";
```

Comment:
maybe remove the word 'limit' here? That would make the messages a bit more clear.